tests/phpunit/tests/blocks/register.php

```diff
@@ -957,6 +957,37 @@ public function data_register_block_registers_with_args_override_returns_false_w
 		);
 	}
 
+	/**
+	 * Tests registering a block with variations from a PHP file.
+	 *
+	 * @ticket 61280
+	 *
+	 * @covers ::register_block_type_from_metadata
+	 */
+	public function test_register_block_type_from_metadata_with_variations_php_file() {
+		$filter_metadata_registration = static function ( $metadata ) {
+			$metadata['variations'] = 'variations.php';
```

Comment:
Should we promote using the `file:` prefix for consistency?

```suggestion
			$metadata['variations'] = 'file:variations.php';
```

It initially came from block assets – CSS and JS file that devs can define in two forms:
- file path: `file:index.js`
- script/style handle: `my-registered-script`

In this case we don't need the distinction for now, but maybe we should make it more strict.

Reply:
Yeah, makes sense! Turns out the `file:` prefix is also used by plenty of blocks in their `render` fields: https://github.com/search?q=repo%3AWordPress%2Fgutenberg%20%22file%3A.%2Frender.php%22&type=code

Reply:
9a30fb28f6

Reply:
Excellent 👍